tpcx_bb/xbb_tools/utils.py

```diff
@@ -790,12 +790,11 @@ def build_benchmark_googlesheet_payload(config):
             if read_graph_creation_time
             else "NA",
             "Machine Setup": data.get("hostname"),
+            "Number of GPUs": data.get("num_workers"),
             "Data Location": data.get("data_dir"),
-            "Repartition_small_table": data.get("repartition_small_table"),
-            "Result verified": data.get("result_verified"),
             "Current Time": current_time,
-            "Device Memory Limit": data.get("device_memory_limit"),
```

Comment:
Did we remove this cause we don't set our cluster up via a cli anymore ?

Reply:
Yeah. We don't really have a clean way of capturing this at the moment, so it's not actually doing anything as is. There are several others like this (such as Infiniband).

I think we can solve all of these in a follow up, and keep the status quo

Reply:
Agreed